normandy/base/api/mixins.py

```diff
@@ -0,0 +1,14 @@
+from django.conf import settings
+from django.views.decorators.cache import cache_control
+
+
+class CachingMixin:
```

Comment:
Let's go with keeping `(object)` then. Removes confusion for Python 2 devs used to inheriting from it coming in to the codebase.